tests/test_install.py

```diff
@@ -26,33 +26,41 @@ def test_install_convert2rhel(
             return_value=pkg_installed_mock,
         ) as mock_run_pkg_check:
             with patch(
-                "scripts.c2r_script._get_last_yum_transaction_id", return_value=1
-            ) as mock_transaction_get:
-                should_undo, _ = install_convert2rhel()
+                "scripts.c2r_script.setup_convert2rhel",
+                return_value=pkg_installed_mock,
+            ) as mock_download_files:
+                with patch(
+                    "scripts.c2r_script._get_last_yum_transaction_id", return_value=1
+                ) as mock_transaction_get:
+                    should_undo, _ = install_or_update_convert2rhel([])
```

Comment:
At some point in the future it would be nice to figure out a way to reduce this nested with call... It is looking pretty scary